### DISCUSSION TITLE
Agency Theme

### DISCUSSION THREAD

**Discussion by Vinaysvinays** [OrchardCMS/OrchardCore#17085] [General]
In agency theme there is a default inbuilt contact us page if we fill that contact us where will be data stored in the database?
![Screenshot 2024-11-26 164903](https://github.com/user-attachments/assets/a673b84f-cb32-4015-bd04-238516d3ba83)

**Comment by daveblack101** [comment]
I think you need to create that logic by yourself. As an example, you can create a workflow that captures the form entries and saves this as a new content item. The steps would be:
- Create a Content Type called 'Enquiries'.
- Add fields Name, Email, Phone and Message
- Create a workflow that runs on form submission
- Add a create content task that saves the form data
